backend/flask_api.py

```diff
@@ -199,5 +199,55 @@ def search():
         return (str(e), 400)
 
 
+# Search function which gets a text and does a full text search on all attributes
+@app.route('/api/projects/search/simple/<int:offset>/<int:count>/', methods=['GET'])
```

Comment:
These functions are already part of another branch AND are outdated. Please remove